EIPS/eip-2917.md

```diff
@@ -0,0 +1,165 @@
+---
+eip: 2917
+title: Staking Reward Calculation
+author: Tony Carson <tony.carsonn@gmail.com>, Mehmet Sabir Kiraz <m.kiraz@gmail.com>, Süleyman Kardaş <skardas@gmail.com>
+discussions-to: https://github.com/ethereum/EIPs/issues/2917
```

Comment:
Discussion-to cannot link to the original PR itself.  You can create a GitHub issue for discussion, or you can create a thread on Ethereum Magicians or somewhere else.